terraform/staging.tfvars

```diff
@@ -36,8 +36,9 @@ website_managed_waf_rules = {
   }
 }
 website_feature_flags = {
-  useNewTable      = true,
-  myProfileEnabled = true
+  useNewTable           = true,
+  myProfileEnabled      = true,
+  newTerminologyEnabled = false
```

Comment:
Per our [docs on deploying feature flags](https://github.com/usdigitalresponse/usdr-gost/blob/_staging/docs/feature-flags.md#deploying):
> a feature that should be available in our for colllaborative QA activities in our Staging environment should be set to an "active" value that enables the new behavior in `staging.tfvars`. Do note that features which are in a highly unstable phase of development or would otherwise be generally disruptive to Staging environment activities should be set to an inactive value.

Given that, asking @ClaireValdivia: should this PR go ahead and enable the terminology changes for Staging once it's merged, or should we hold off on that for now?